editor/src/components/canvas/controls/size-box.tsx

```diff
@@ -210,6 +297,12 @@ const ResizeLines = (props: ResizeLinesProps) => {
     props.resizeStatus !== 'enabled' ? null : (
       <div
         ref={reference}
+        onMouseEnter={() => {
+          setShowLabel(true)
+        }}
+        onMouseLeave={() => {
+          setShowLabel(false)
+        }}
```

Comment:
`React.useCallback`

Reply:
Fixed in 5b68dd64a689f218fbe4e203d4db7b5a9433a636.